docs/design/bundle-sources.md

```diff
@@ -0,0 +1,300 @@
+
+        ```sh
+        .
+        ├── <catalog_dir>
+        └── <catalog_dir>.Dockerfile
+        ```
+
+1. Populate the catalog with the package definition for your Operator by running the `opm init` command:
+
+    ```sh
+    opm init <operator_name> \
+    --default-channel=preview \
+    --description=./README.md \
+    --icon=./operator-icon.svg \
+    --output json \
+    > <catalog_dir>/index.json
+    ```
+
+    This command generates an `olm.package` declarative config blob in the specified catalog configuration file.
+
+## Adding a plain bundle to your file-based catalog
+
+Currently, the `opm render` command does not support adding plain bundles to catalogs. You must manually add plain bundles to your file-based catalog, as shown in the example below.
+
+<h3 id="prereqs-adding-a-plain-bundle-to-fbc">
+Prerequisites
+</h3>
+
+* `opm` CLI tool
+* A plain bundle image
+* A file-based catalog
+* Push access to an image registry, such as [Quay](https://quay.io)
+* Docker or Podman
+
+<h3 id="proc-adding-a-plain-bundle-to-fbc">
+Procedure
+</h3>
+
+1. Verify that your catalog's `index.json` or `index.yaml` file is similar to the following example:
+
+    *Example `<catalog_dir>/index.json` file*
+
+    ```json
+    {   
+        {
+         "schema": "olm.package",
+         "name": "<operator_name>",
+         "defaultChannel": "<default_channel_name>"
+        }    
+    }
+    ```
+
+1. To create an `olm.bundle` blob, edit your `index.json` or `index.yaml` file, similar to the following example:
+
+    *Example `<catalog_dir>/index.json` file*
+
+    ```json
+    {
+        "schema": "olm.bundle",
+        "name": "<bundle_name>",
+        "package": "<operator_name>",
+        "image": "quay.io/<image_repo>/plainbundle:test",
+        "properties": [
+            {
+                "type": "olm.package",
+                "value": {
+                "packageName": "<operator_name>",
+                "version": "0.0.3"
+                }
+            },
+            {
+                "type": "olm.bundle.mediatype",
+                "value": "plain+v0"
+            }
+      ]
+    }
+    ```
+
+1. To create an `olm.channel` blob, edit your `index.json` or `index.yaml` file, similar to the following example:
+
+    *Example `<catalog_dir>/index.json` file*
+
+    ```json
+    {
+        "schema": "olm.channel",
+        "name": "<default_channel_name>",
+        "package": "<operator_name>",
+        "entries": [
+            {
+                "name": "<bundle_name>"
+            }
+        ]
+    }
+    ```
+
+<h3 id="verify-adding-a-plain-bundle-to-fbc">
+Verification
+</h3>
+
+1. Open your `index.json` or `index.yaml` file and ensure it is similar to the following example:
+
+    *Example `index.json` file*
+
+    ```json
+    {
+        "schema": "olm.package",
+        "name": "testOperator",
+        "defaultChannel": "preview"
+    }
+    {
+        "schema": "olm.bundle",
+        "name": "plainbundle:test",
```

Comment:
In OLMv0, the standard convention for bundle name is: `<packageName>.v<bundleVersion>`. I'd suggest following that same standard, for now at least (this is another area of FBC that we may want to do something different with in the future)

Reply:
Good point, modified it to adhere to the standard convention